src/event-handler/remove-listener.ts

```diff
@@ -2,7 +2,7 @@ import { EmitterWebhookEventName, State } from "../types";
 
 export function removeListener(
   state: State,
-  webhookNameOrNames: EmitterWebhookEventName | EmitterWebhookEventName[],
+  webhookNameOrNames: '*' | EmitterWebhookEventName | EmitterWebhookEventName[],
```

Comment:
This differs from the original proposed fix in that I omit the following type union:

```
| (EmitterWebhookEventName | "*")[]
```

Which I think is fine here, as you want to remove everything OR individual event names. I don't see a use case for ever specifying both like:

```
webhooks.removeListener(["check_run.created", "*"], ...
```